dbt/adapters/firebolt/impl.py

```diff
@@ -313,7 +313,8 @@ def get_rows_different_sql(
             names = sorted((self.quote(n) for n in column_names))
 
         where_expressions = [
-            f'{relation_a}.{name} = {relation_b}.{name}' for name in names
+            f'{relation_a}.{name} IS NOT DISTINCT FROM {relation_b}.{name}'
+            for name in names
```

Comment:
This is the best we can do before EXCEPT is implemented, as previous query has been made PG-compliant and no longer works as we expect. Discussed [here](https://firebolt-analytics.slack.com/archives/CU5A6DWAE/p1753448082434429?thread_ts=1753447056.463309&cid=CU5A6DWAE).